r124/outref/ydb312_gtm8182g.txt

```diff
@@ -0,0 +1,43 @@
+Create the DB
+
+
+%GDE-I-GDUSEDEFS, Using defaults for Global Directory 
```

Comment:
GDE outputs don't need to be in reference file. Instead a line saying GBL1.gld and GBL2.gld are created to point to mumps.dat and a.dat respectively is enough. Also indicate both gld files point to mumps.repl instance file.